### PR TITLE
Implement function to return statistics of returning visitor sales

### PR DESCRIPTION
**Pull request by hrxiao**
Resolving Issue #24

### CODE REVIEW

File: explore_utils.py [hunk 1]
```diff
@@ -120,3 +120,27 @@ def find_transaction_by_region(data):
     new_df = avg[avg['rev']>0]
     new_df.columns = ['Transaction']
     return new_df.sort_values(by=['Transaction'])
+
+
+def find_return_visit_stats(dataset):
+    """Find the statistics of total transactions for returnining visitors
```
Comment (sammysrg) [inline explore_utils.py]: returning not returnining

File: explore_utils.py [hunk 2]
```diff
@@ -120,3 +120,27 @@ def find_transaction_by_region(data):
...
+    group_df = (train_df[['fullVisitorId', 'visitNumber', 'revenue']]
+          .groupby('fullVisitorId', as_index=False)
+          .agg({'visitNumber': 'max', 'revenue': 'sum'}))
+    first_stats = (group_df[group_df['visitNumber'] == 1]['revenue']
+                  .describe(percentiles=[.95, .99, .999, .9999])
+                  .rename('First Time Visitor'))
+    return_stats = (group_df[group_df['visitNumber'] != 1]['revenue']
+                    .describe(percentiles=[.95, .99, .999, .9999])
+                    .rename('Return Visitor'))
+    return pd.concat([first_stats, return_stats], axis=1)
```
Comment (sammysrg) [inline explore_utils.py]: perhaps add a line that gets rid of the extra decimal points. The counts can be changed to integers while the other statistics should be integers or have 2 decimal points.

Comment (jonathancstroud) [reply explore_utils.py]: It's probably easier to do the reformatting in `explore.py` when you print the numbers out. Also, this way the function doesn't decide how to round, instead the user decides how many decimal points to round.

Comment (hrxiao) [reply explore_utils.py]: Added a rounding before printing in `explore.py` so the accuracy of the statistics can be preserved.

File: explore_utils.py (hunk 1 above)
Comment (hrxiao) [reply explore_utils.py]: Typo fixed

File: explore_utils.py (hunk 2 above)
Comment (jonathancstroud) [reply explore_utils.py]: Looks great!

**Pull request merged**